vision_agent/agent/vision_agent_prompts.py

```diff
@@ -179,6 +179,8 @@ def find_text(image_path: str, text: str) -> str:
 8. DO NOT use try except block to handle the error, let the error be raised if the code is incorrect.
 9. DO NOT import the testing function as it will available in the testing environment.
 10. Print the output of the function that is being tested.
+11. Use the output of the function that is being tested as the return value of the testing function.
+12. Run the testing function in the end and don't assign a variable to its output.
```

Comment:
Trying to think of ways to keep the list of instructions shorter as I suspect as the list gets longer the ability to follow the directions get's poorer.

Would this also work if we removed instruction 10. (as it seems to accomplish the same thing as 11.) and combine 11. and 12. into "Return the output of the function that is being tested in the test script, do not assign it to another variable" or something shorter?

Reply:
I tried a few approaches. This actually is the best prompt so far.
Other approaches significantly increase the chance of the testing function return null.
Here are approaches i have tried:
1. Removing #10
2. Merge 11 + 12 into one instruction
3. Rewrite 11 in a more concise way
4. Move 11 + 12 to the 3th and 4th instruction
5. Move all instructions up

And some combinations of the above.